### ISSUE TITLE
WHERE Clause not working on enum type

### ISSUE THREAD

**Issue by ihsanmohamad** [electric-sql/electric#1662]
Hey there, I'm running into an issue with the Electric-sql, and I've put together a minimal reproduction using Next.js to help illustrate the problem. I've included the schema for reference.

For some context, my main project is built with SvelteKit, and I initially thought the issue might be related to my Svelte implementation of the electric-sql TypeScript client. However, after creating this Next.js example, it seems the problem might be on enum columns.

Basically, any column defined as an enum in the schema is causing issues, while other column types are working just fine. I'm not entirely sure what's going on, but I wanted to provide this reproduction to see if anyone has encountered a similar problem or has any insights to share.

[https://github.com/ihsanmohamad/react-electric-repro](https://github.com/ihsanmohamad/react-electric-repro)

**Comment by KyleAMathews** [comment]
Hey, sorry you're running into issues. Could you modify your README with instructions on how to run the reproduction? Ideally a single command can e.g. start a docker compose file and run the migration. I don't know how to run your app as is.

**Comment by ihsanmohamad** [comment]
ok, will do tomorrow.

**Comment by ihsanmohamad** [comment]
hi I updated the repo. I also add the README with instructions. I'm sorry I am unable to make fully docker compose as I'm not familiar with docker and nextjs. I'll try to work on docker compose again if you have issues with the steps.

**Comment by ihsanmohamad** [comment]
> We got a similar [bug report on Discord](https://discord.com/channels/933657521581858818/933657523049885698/1283001101536465001) yesterday. Could this be related?

that was me 😁
